serde-generate/runtime/csharp/Serde.Tests/Serde.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.1</TargetFramework>
+    <TargetFramework>netcoreapp2</TargetFramework>
```

Comment:
I had to remove the . so the tests pass on my laptop with netcoreapp 3.1 (still lots of warnings but at least it works)

Reply:
That's fine. I didn't choose this for any specific reason AFAIR, it's just what I had installed.